src/BeamAdapter/component/controller/InterventionalRadiologyController.inl

```diff
@@ -788,7 +788,7 @@ void InterventionalRadiologyController<DataTypes>::applyInterventionalRadiologyC
     Data<VecCoord>* datax = this->getMechanicalState()->write(core::VecCoordId::position());                
     auto x = sofa::helper::getWriteOnlyAccessor(*datax);
 
-    VecCoord xbuf =x;
+    VecCoord xbuf = x.ref();
```

Comment:
This results in a copy, right?

Reply:
yes, it was the case before but for some reason, it did not like the =operator with the accessor